packages/fcl/src/current-user/index.js

```diff
@@ -94,21 +87,24 @@ async function authenticate() {
 
     const discoveryWallet = await config.first([
       "discovery.wallet",
-      "challenge.handshake"
+      "challenge.handshake",
     ])
 
     try {
       if (discoveryWallet == null) {
         console.warn(
           `Required value for "discovery.wallet" not defined in config. See: ${"https://github.com/onflow/flow-js-sdk/blob/master/packages/fcl/src/exec/query.md#configuration"}`
         )
-        throw new Error("Required Config Value Not Defined")
+        throw new Error("config.discovery.wallet is not defined")
```

Comment:
Maybe something like:

```suggestion
        throw new Error(`Required config value "config.discovery.wallet" is not defined`)
```

Just to make it clear that its about a config variable